snooty/test_openapi.py

```diff
@@ -21,15 +21,15 @@ def test_openapi_with_snooty() -> None:
         path,
         """
 .. openapi:: /test_parser/openapi-admin-v3.yaml
-   :snooty-parse:
+   :uses-rst:
```

Comment:
I think this ended up backwards somehow—the Realm OpenAPI spec uses rST and the Cloud specs going forward will not. So we probably want to use the flag in the first test, sorry for the confusion Raymund!